frontend/pages/login.vue

```diff
@@ -102,6 +110,20 @@ export default {
           : this.error;
       }
     },
+    authToken() {
+      const rawAuthToken = this.$route.query.auth;
+
+      if (!rawAuthToken) return;
+
+      try {
+        const auth = JSON.parse(atob(rawAuthToken));
+
+        if (!!auth.username && !!auth.password) return auth;
+      } catch {}
```

Comment:
Please fix `Empty block statement` error

Reply:
Hi Leire, The error in this case is not important, as the token can have any value.

Reply:
Is because the python package build is failing

Reply:
@damianpumar https://github.com/argilla-io/argilla/actions/runs/5350353696/jobs/9702843904?pr=3253